### PR TITLE
Updated with devtool gem.

### PR DESCRIPTION
**Pull request by jweakley**

### CODE REVIEW

**Comment by jweakley** [comment]
@Samsinite or @grelement can you review?

File: template.rb [hunk 1]
```diff
@@ -8,8 +8,8 @@
 # This template assumes you have followed the setup guide at:
 # https://github.com/wildland/guides/#setting-up-your-development-enviroment
 #
-ruby_version = '2.1.5'
-node_version = 'iojs-v1.5.0'
+ruby_version = '2.2.2'
+node_version = 'v0.12.7'
```
Comment (Samsinite) [inline template.rb]: There is v4.0 now, should we use it instead?

File: template.rb [hunk 2]
```diff
@@ -49,18 +45,8 @@
 gsub_file 'gemfile', /gem 'rails', '/, "gem 'rails', '~>"
 
 # kill un-needed gems
-run "sed -i.bak '/turbolinks/d' Gemfile"
-
-run "sed -i.bak '/coffee/d' Gemfile"
 
 run "sed -i.bak '/jbuilder/d' Gemfile"
-run "sed -i.bak '/jquery-rails/d' Gemfile"
```
Comment (Samsinite) [inline template.rb]: Does the rails command above remove jquery as well?

File: template.rb [hunk 3]
```diff
@@ -239,7 +225,7 @@ def index
 run 'rails g api_me:install'
 
 # Token Authentication Installation and Setup (token_authenticate_me and ember-authenticate-me)
-gem 'token_authenticate_me'
+gem 'token_authenticate_me', '>=0.4.2'
```
Comment (Samsinite) [inline template.rb]: Do we want to pin this down? Then it won't install the latest by default.

**Comment by Samsinite** [comment]
:+1: after addressing feedback.

File: template.rb (hunk 2 above)
Comment (jweakley) [inline template.rb]: The `-J` flag removes all of this.

File: template.rb (hunk 3 above)
Comment (jweakley) [inline template.rb]: As far as I know, it should try to pull the most recent version that matches. This was the one I tested with so I assumed it or higher would work.

File: template.rb (hunk 1 above)
Comment (jweakley) [inline template.rb]: I was using normal node instead of io.js. Should we switch to that?

File: template.rb (hunk 2 above)
Comment (Samsinite) [inline template.rb]: Sweet. missed that flag.

File: template.rb (hunk 3 above)
Comment (Samsinite) [inline template.rb]: Sorry, missed the > symbol.

File: template.rb (hunk 1 above)
Comment (Samsinite) [inline template.rb]: Referring to normal nodejs, v4 was the release that includes the merge of iojs into node: https://nodejs.org/en/blog/release/v4.0.0/.

**Pull request merged**